### PR TITLE
Fix relative import of `base_test` in test files

### PR DESCRIPTION
**Pull request by mayaCostantini**
## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Fixing relative imports of `base_test` in test files.

### CODE REVIEW

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
can you check on the pytest ?

File: .prow.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ presubmits:
       containers:
         - image: quay.io/thoth-station/thoth-pytest-py38:v0.12.8
           command:
-             - "/bin/run-pytest"
+             - pytest
```
Comment (goern) [inline .prow.yaml]: This should not be changed, because it changes the interface how we want pytest to be invoced.

**Review by goern — CHANGES_REQUESTED** [review]

File: .prow.yaml (hunk 1 above)
Comment (mayaCostantini) [reply .prow.yaml]: Fixed

**Pull request merged**